packages/ai-jsx/src/core/completion.tsx

```diff
@@ -12,6 +11,14 @@ import { getEnvVar } from '../lib/util.js';
 import { AnthropicChatModel } from '../lib/anthropic.js';
 import z from 'zod';
 import { zodToJsonSchema } from 'zod-to-json-schema';
+export {
+  UserMessage,
+  SystemMessage,
+  AssistantMessage,
+  FunctionCall,
+  FunctionResponse,
+  ConversationHistory,
+} from './conversation.js';
```

Comment:
Presumably you're doing this to avoid breaking backwards compat, but also think that `conversation.js` is the more natural home for these? In that case, we should mark these as `@deprecated` and give a note telling people to use `conversation` instead.